src/Symfony/Component/Serializer/Tests/Mapping/Loader/AnnotationLoaderTest.php

```diff
@@ -123,6 +123,14 @@ public function testLoadContexts()
         $this->assertLoadedContexts($this->getNamespace().'\ContextDummy', $this->getNamespace().'\ContextDummyParent');
     }
 
+    /**
+     * @requires PHP 8
```

Comment:
This `@requires` won't be necessary if we merge this to 6.2

Reply:
Feel free to merge it either in 4.4 (as i saw this is latest maintained version) or 6.2. Or do you need me to update this here?

Reply:
The attribute is not available in 4.4 yet.
Since @dunglas seems to agree with me this should be considered a new feature, as of PHP 8 specific feature, we'll merge it to 6.2. The rebase and changing this line is not necessary, we can do it while merging, but updating the PR to the right branch might be helpful to detect unexpected issues on the CI.